src/documents/CoreAPI.md

```diff
@@ -146,6 +149,14 @@ globally unique with high probability (50% chance of collision after a million t
 `prefix` is the value of `fluid.fluidInstance` identifying this Infusion instance, and `id` is a increasing integer initialised at
 1 when this infusion instance starts up.
 
+### fluid.add(a, b)
```

Comment:
The implementation of `fluid.add()` doesn't check whether `a` and `b` are numbers. Is it callers' responsibility to make sure they are numbers? Otherwise this method is also good for string concatenations etc. :)